ocaml/idl/datamodel_common.ml

```diff
@@ -10,7 +10,7 @@ open Datamodel_roles
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
 
-let schema_minor_vsn = 764
+let schema_minor_vsn = 765
```

Comment:
Master is on 762 - so I'm surprised by 764 and 765.

Reply:
OK, will revert the change here, and increase the version by 1 when the feature branch is merged into master.